src/proof/alethe/alethe_post_processor.cpp

```diff
@@ -86,6 +87,14 @@ bool AletheProofPostprocessCallback::update(Node res,
     // node by using an extra pair of parenthesis, i.e. ((or b c)) is the proof
     // node printed as (cl (or b c)).
     //
+    // Adding an OR node to a premises will take place in the finalize function
+    // where in the case that a step is printed as (cl (or F1 ... Fn)) but used
+    // as (cl F1 ... Fn) an OR step is added to transform it to this very thing.
+    // This is necessary for rules that work on clauses, i.e. RESOLUTION,
+    // CHAIN_RESOLUTION, REORDERING and FACTORING. Thus, for all these rules the
+    // reader can assume that the premise was printed correctly since if not it
+    // is after the pass of the proof node in finalize.
```

Comment:
```suggestion
    // CHAIN_RESOLUTION, REORDERING and FACTORING.
```
I think it's more clear if we omit the last sentence.